src/Microsoft.Health.Fhir.Shared.Api/Modules/OperationsModule.cs

```diff
@@ -62,10 +60,10 @@ public void Load(IServiceCollection services)
 
             services.AddSingleton<IReindexUtilities, ReindexUtilities>();
 
-            services.Add<OperationsCapabilityProvider>()
+            /*services.Add<OperationsCapabilityProvider>()
```

Comment:
>OperationsCapabilityProvider [](http://example.com/codeflow?start=27&length=28)

I don't think it should be changed and it's related to graphql.

Reply:
While building, I have a compile error, so I changed this with the objective to do it buildable.